Libplanet/Net/NetMQTransport.cs

```diff
@@ -749,7 +766,12 @@ private async Task RefreshAllocate(CancellationToken cancellationToken)
 
         private async Task ProcessRequest(MessageRequest req, CancellationToken cancellationToken)
         {
-            _logger.Verbose("Request {RequestId} taken.", req.Id);
+            _logger.Verbose(
+                "Request {Message}({RequestId}) is ready to be processed in {TimeSpan}ms.",
```

Comment:
ASFAIK, [`TimeSpan.ToString()`][1] automatically formats its subseconds as well:

```csharp
> new TimeSpan(days: 0, hours: 1, minutes: 2, seconds: 3, milliseconds: 4).ToString()
"01:02:03.0040000"
```

```suggestion
                "Request {Message}({RequestId}) is ready to be processed in {TimeSpan}.",
```

[1]: https://docs.microsoft.com/en-us/dotnet/api/system.timespan.tostring?view=netstandard-2.0